src/kibana/plugins/vis_types/vislib/editors/tile_map.html

```diff
@@ -10,8 +10,36 @@
   </select>
 </div>
 
+<div ng-if="vis.params.mapType === 'Heatmap'">
+  <div class="vis-editor-agg-form-row">
+    <div class="form-group">
+      <label>Radius</label>
+      <input name="heatRadius" ng-model="vis.params.heatRadius" required class="form-control ng-valid ng-valid-number ng-valid-min ng-valid-required ng-dirty" type="number" step="1" min="5" max="50">
```

Comment:
Whats with all the ng-X classes attached here? These are usually handled by angular.